frontend/public/extend/devconsole/components/Pipelines/PipelineDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {
+  SectionHeading,
+  ResourceSummary
+} from '../../../../components/utils';
+
+//const TaskRow = ({obj: task}) =><><dd>{task.name}</dd><dt>Ref: {task.taskRef.name}</dt></>;
+
+const DevPipelineDetails = ({obj:pipeline}) => <div className="co-m-pane__body">
```

Comment:
Use `(` for multi-line JSX. 
```suggestion
const DevPipelineDetails = ({obj:pipeline}) => (
  <div className="co-m-pane__body">
```